en/routing.texy

```diff
@@ -166,7 +166,10 @@ $router[] = new Route('<presenter>/<action>[/<id>]', 'Homepage:default');
 .[note]
 Unlike other frameworks, Nette does not require routes to be named.
 
-It's important in which order are the routes defined as they are tried from top to bottom. The rule of thumb here is that routes are declared from the **most specific at the top** to the **most vague at the bottom**. Keep in mind that huge amount of routes can negatively effect application speed, mostly when generating links. It's worth to keep routes as simple as possible.
+.[caution]
+It's important in which order are the routes defined as they are tried from top to bottom. The rule of thumb here is that routes are declared from the **most specific at the top** to the **most vague at the bottom**.
+
+Keep in mind that huge amount of routes can negatively effect application speed, mostly when generating links. It's worth to keep routes as simple as possible.
```

Comment:
*effect* -> *affect*
*worth keeping* nebo *worth noting that routes should be kept ...*